src/lib/charm/openstack/infoblox.py

```diff
@@ -1,13 +1,11 @@
-#!/usr/bin/python2
+#!/usr/bin/python3
```

Comment:
This line isn't needed, and can be removed.  As this isn't an executable script with the equivalent of a main, it just looks a bit confusing as this is a module rather than a script to be called directly.

Reply:
Removed.